src/enums/index.ts

```diff
@@ -12,7 +12,8 @@ export enum EmailStatusTypes {
 export enum ApplicationStatus {
```

Comment:
Shall we rename this to `MenteeStatus` and create separate one form Mentors? Cuz mentors don't need a completed state. 

```suggestion
export enum MenteeStatus {
```

Reply:
For now, could we avoid changing this? There is a lot of refactoring involved.